### DISCUSSION TITLE
ImGui Fatal Error upon injection

### DISCUSSION THREAD

**Discussion by JakeRuCodes** [ocornut/imgui#6801] [New Users Build/Link/Run issues ONLY!]
I tried implementing ImGui::Image the way ocornut provided and to no avail I get a fatal error, it builds just fine with no errors but isn't working? Some help would be nice, I will provide my source.

Rendering: DX11 - Kiero Hook
Build: Release (x64)
Compiler: VS 2022 Community
OS: Windows
ImGui: 1.89.9

**Comment by ocornut** [comment]
Sorry but #1586
